src/commands/lambda/__tests__/instrument.test.ts

```diff
@@ -1361,43 +1189,43 @@ describe('lambda', () => {
       })
 
       test('prints which functions failed to instrument without aborting when at least one function was instrumented correctly', async () => {
+        ;(fs.readFile as any).mockImplementation((a: any, b: any, callback: any) => callback({code: 'ENOENT'}))
         const failingLambdas = [
           'arn:aws:lambda:us-east-1:123456789012:function:lambda-1-us-east-1',
           'arn:aws:lambda:us-east-1:123456789012:function:lambda-2-us-east-1',
           'arn:aws:lambda:us-east-2:123456789012:function:lambda-1-us-east-2',
         ]
-        ;(fs.readFile as any).mockImplementation((a: any, b: any, callback: any) => callback({code: 'ENOENT'}))
         mockLambdaConfigurations(lambdaClientMock, {
           'arn:aws:lambda:us-east-1:123456789012:function:lambda-1-us-east-1': {
             config: {
               FunctionArn: 'arn:aws:lambda:us-east-1:123456789012:function:lambda-1-us-east-1',
               FunctionName: 'lambda-1-us-east-1',
               Handler: 'index.handler',
-              Runtime: 'nodejs22.x',
+              Runtime: 'nodejs16.x',
```

Comment:
Result of this move: https://github.com/DataDog/datadog-ci/pull/1622#discussion_r2053858570